### PR TITLE
docs: Update add options text (no-changelog)

### PR DESCRIPTION
**Pull request by Joffcom**
## Summary
Adds placeholder text to the button so it appears

image

Also updates all `Add Options` to be `Add option` for consistency, See: https://n8nio.slack.com/archives/C06K5TQRJJD/p1721039761152119 for more info from @gandreini

## Related Linear tickets, Github issues, and Community forum posts
https://linear.app/n8n/issue/NODE-1501/options-not-visible-in-slack-node

### CODE REVIEW

File: packages/nodes-base/nodes/Slack/V2/MessageDescription.ts [hunk 1]
```diff
@@ -1223,12 +1223,12 @@ export const messageFields: INodeProperties[] = [
...
 					'Choose from the list, or specify IDs using an <a href="https://docs.n8n.io/code-examples/expressions/">expression</a>',
 				default: [],
-				placeholder: 'Select a channel...',
 				typeOptions: {
 					loadOptionsMethod: 'getChannelsName',
 				},
 			},
 		],
 		default: {},
+		placeholder: 'Add options',
```
Comment (ShireenMissi) [inline packages/nodes-base/nodes/Slack/V2/MessageDescription.ts:1232]:
```suggestion
		placeholder: 'Add Option',
```

**Review by ShireenMissi — DISMISSED** [review]

**Review by ShireenMissi — DISMISSED** [review]
LGTM 🎉

**Comment by Joffcom** [comment]
Ahhh I just created a merge conflict, I will fix that shortly.

**Comment by Joffcom** [comment]
@ShireenMissi I have fixed the merge conflict i created, Would you be able to approve again?

**Review by ShireenMissi — APPROVED** [review]
LGTM 🎉

**Comment by janober** [comment]
Got released with [`n8n@1.53.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.53.0)

**Pull request merged**